src/encryption/api.go

```diff
@@ -0,0 +1,75 @@
+package encryption
+
+import (
+	"context"
+	"fmt"
+	"keboola-as-code/src/client"
+	"strings"
+
+	"github.com/go-resty/resty/v2"
+	"go.uber.org/zap"
+)
+
+type EncryptionApi struct {
+	apiHost string
+	client  *client.Client
+	logger  *zap.SugaredLogger
+}
+
+// Error represents Encryption API error structure
+type Error struct {
+	Message     string `json:"error"`
+	ErrCode     int    `json:"code"`
+	ExceptionId string `json:"exceptionId"`
+}
+
+func (e *Error) Error() string {
+	msg := fmt.Sprintf(`"%v", errCode: "%v"`, e.Message, e.ErrCode)
+	if len(e.ExceptionId) > 0 {
+		msg += fmt.Sprintf(`, exceptionId: "%s"`, e.ExceptionId)
+	}
+	return msg
+}
```

Comment:
formatovanie chyby som nejak extra neriesil, proste sa to vypise podla struktury Error